### DISCUSSION TITLE
Is open telemetry for .net working?

### DISCUSSION THREAD

**Discussion by ghost** [open-telemetry/community#610] [Community Discussions]
The reason for me asking is that I have spent days now trying to get a proof of concept with Grafana Tempo and OpenTelemetry running with my .net core application and I have tried everything now but with no progress.

I have followed this guide (https://grafana.com/docs/tempo/latest/getting-started/?pg=oss-tempo&plcmt=hero-txt) and my docker images is up and running as they should. However it seems that my problem is that no traces whatsoever are exported by my Jaeger exporter to my tempo docker image running on port 6831 as in their guide. As you can see below is my setup of opentelemetry and the AddJaegerExporter() part is using localhost and 6831 as default values so I expected to be all set with that. However, the only thing my tempo docker image logs are showing is startup info and no incoming traces of any kind. I do not see any information on traces being sent in application logs either. When activating console exporter I see those traces in application logs but why isn't jaeger exporter working? Have tried zipkin and opentelemetry as well without any result.

services.AddOpenTelemetryTracing(builder => builder
					.SetResourceBuilder(ResourceBuilder.CreateDefault()
						.AddService("Authentication API"))
					.AddAspNetCoreInstrumentation()
					.AddHttpClientInstrumentation()
					.AddJaegerExporter()

If I try a search in the querycontainer for a traceId I obviously get nothing since nothing seems to be transferred to the tempo container. However, I get an error in the logs saying:
{"level":"error","ts":1611087947.883616,"caller":"zap/logger.go:33","msg":"failed to flush Jaeger spans to server: write udp 127.0.0.1:37973->127.0.0.1:6831: write: connection refused","stacktrace":"github.com/uber/jaeger-client-go/log/zap.(*Logger).Error\n\tgithub.com/uber/jaeger-client-go@v2.23.1+incompatible/log/zap/logger.go:33\ngithub.com/uber/jaeger-client-go.

Tempo isn't flushing traces when I search is it? Shouldn't matter though since jaeger exporter seems to not export anything. Where 127.0.0.1:37973 comes from I have no Idea. My application is configured to send traces to tempo:6831. I

**Comment by SergeyKanzhelev** [comment]
CC: @open-telemetry/dotnet-approvers 

Thank you for your question. Do you have a test project that is not working with Jaeger?
